Lombiq.Hosting.Tenants.Management.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -19,10 +19,18 @@ public static async Task TestShellSettingsEditorFeatureAsync(this UITestContext
         await context.FillInEditorThenCheckValueAsync(
             "{\"TestKey\":{\"TestSubKey\":{\"TestSubOptions\":{\"Test\": \"TestValue\"}}}}",
             "TestValue");
+
+        await context.FillInEditorThenCheckValueAsync(
+            "{\"TestKey\":{\"TestSubKey\":{\"TestSubOptions\":{\"NewKey\": \"NewValue\"}}}}",
+            "NewValue");
+
+        await context.FillInEditorThenCheckValueAsync(
+            "{\"TestKey\":{\"TestSubKey\":{\"TestSubOptions\":{}}}}",
```

Comment:
Should this really zero out the editor? Also, test removing a key in a way that other keys remain, not removing the whole config (that's the next test).

Reply:
> Should this really zero out the editor?

Yes this should because there won't be any key value pairs anymore.

>  (that's the next test).

okay, adding new test cases then.